pkg/kubectl/rolling_updater_test.go

```diff
@@ -1413,7 +1413,7 @@ func TestUpdateRcWithRetries(t *testing.T) {
 		{StatusCode: 200, Header: header, Body: objBody(codec, rc)},
 	}
 	fakeClient := &fake.RESTClient{
-		Codec: codec,
+		NegotiatedSerializer: testapi.Default.NegotiatedSerializer(),
```

Comment:
All of these style changes are fine.  We should have a helper for returning a single serializer (and handling defaulting) to make code simpler.